packages/angular/README.md

```diff
@@ -53,19 +51,14 @@ This library is providing an `NgModule` named `SynergyModule`, which takes care
 
 ```typescript
 // src/app/app.module.ts
-import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { AppComponent } from './app.component';
-import { SynergyComponentsModule } from '@synergy-design-system/angular';
+import { NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+import { AppComponent } from "./app.component";
+import { SynergyComponentsModule } from "@synergy-design-system/angular";
 
 @NgModule({
-  declarations: [
-    AppComponent,
-  ],
-  imports: [
-    BrowserModule,
-    SynergyComponentsModule,
-  ],
+  declarations: [AppComponent],
+  imports: [BrowserModule, SynergyComponentsModule],
```

Comment:
Same for here, all Angular Projects come with an imports section. The multi line syntax makes it easier to see and copy the needed string, so I was opting for this to be used.

Reply:
We need some global linting rules for Markdown, YAML, JSON etc. 

Apparently our local rules clash sometimes with each other. I don't see this as part of this ticket, but as something coming up several times now.

Suggestion: Prettier for all those files from root and done.

Reply:
I personally do not use code-formatting /beautifying at all because of such issues. Also, it may make some sense for denser syntax when really working with code, but when working with documentation I want extensive and easy to read (if not to say exhaustive) documentation for the parts that matter. When using automatic formatting, you take this choice away to the tool that you are using.

We have to think about how to handle this later on.

Reply:
I don't think we should lose any time on discussing code formatting. I'm fine with any kind of code formatting as long as it's in any way doable automatically. Take Prettier or whatever. But we need coded rules. As long as there aren't any in place, we can't expect any one in the team to handle it according to personal preferences. And discussing linebreaks, spacings etc. is cumbersome and always personal – that's why I use automatic tools for that.

I will open a ticket for that.

Reply:
Done: https://github.com/SickDesignSystem/synergy/issues/181

I resolved your issues to get this PR merged, but please let's address this ASAP.